Remotion/ObjectBinding/Web/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -66,15 +73,24 @@
   <svg class="sprite-part" id="ReferenceValueErrorIndicator" width="16px" height="16px" viewBox="0 0 16 16">
     <use href="#Alert" />
   </svg>
-  <image id="RequiredField" height="17" width="12" href="data:image/gif;base64,R0lGODlhDAARAPcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAAP8ALAAAAAAMABEAAAgtAP8JHEiwoMGDCBMqVAgAwMGG/yBCJNiwokODFh9avDhQosOJBUEuHEmy5MiAADs=" />
+  <svg class="sprite-part" id="RequiredField" height="16px" width="16px" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round">
+      <line x1="8" y1="4" x2="8" y2="12" stroke="#555555"/>
+      <line x1="4" y1="8" x2="12" y2="8" stroke="#555555"/>
+      <line x1="5.17157288" y1="10.8284271" x2="10.8284271" y2="5.17157288" stroke="#555555"/>
+      <line x1="10.8284271" y1="10.8284271" y2="5.17157288" x2="5.17157288" stroke="#555555"/>
+    </g>
+  </svg>
```

Comment:
I think we can drop this one. It's the required marker in BocList colums, which now use the icon from Web instead of ObjectBinding.